binding_generator.py

```diff
@@ -68,11 +70,11 @@ def generate_virtual_version(argcount, const=False, returns=False, required=Fals
     s = """#define GDVIRTUAL$VER($RET m_name $ARG)\\
```

Comment:
why don't you return python's `f"""`string instead? Regexes are less readable, have no IDE support and might be error prone (i.e. four changes from now someone will make typo and it might slip unnoticed).

(that's not important for now thought, it is something that might be tackled as a follow-up task as a `good first issue` or something)

Reply:
and IMO `ret = "m_ret" if returns else ""` might be easier to reason about than separate if/else blocks.

Reply:
Perhaps! However:

1. That's something for a different PR. This code is only touched in this PR due to some renames that were search-and-replaced - the changes you describe would not be related to the high-level goals of this PR
2. This code was copied from `make_virtuals.py` in Godot itself and modified slightly. I don't want it to get too out-of-sync with the original, since we do periodically copy code back from Godot into godot-cpp. So, if we were to change this, ideally, we'd change it in both places